### ISSUE TITLE
Not enough generation capacity - dummy technology to cover rest of load

### ISSUE THREAD

**Issue by jonasVano** [open-modex/models#23]
Hi all,

in our model we have so called "slack-cost". They are applied to the amount of energy that is missing in the system due to shortage in generation. Do you have something similar in your models? If so, do we need a parameter to harmonize those cost? Do you have any uncovered load anayway?

**Comment by khainsch** [comment]
Yes, we do have the functionality of said slack costs (with incredible high costs so they are really only triggered if necessary). But so far, we do not see these being required in the current model runs.

**Comment by jonasVano** [comment]
I would suggest to use a very high value: 999999
